### ISSUE TITLE
playFacialAnim native sync through altv

### ISSUE THREAD

**Issue by tastydev** [altmp/altv#112]
It would be nice to have the playFacialAnim native synced through altv to perform features like lipsync, for example:
if(data.talking){
    game.playFacialAnim(this.localPlayer.scriptID, "mic_chatter", "mp_facial");
                    }else{
    game.playFacialAnim(this.localPlayer.scriptID, "mood_normal_1", "facials@gen_male@variations@normal");
}

**Comment by Revyn112** [comment]
this seems not to be fixed or i interpreted this wrong

on my local client i `run game.playFacialAnim(this.localPlayer.scriptID, "mic_chatter", "mp_facial");`
My Peds start moving like expected, but other players see's my mouth not moving - so it's not synced properly.

**Comment by Lhoerion** [comment]
Does it happen on latest beta build #1004?

**Comment by tastydev** [comment]
Still have to sync it by myself

**Comment by R0ckzo0r** [comment]
Same here

**Comment by martonp96** [comment]
Is there a difference between playFacialAnim and taskPlayAnim? I would suggest you to try taskPlayAnim instead @tastydev .

**Comment by C0kkie** [comment]
The difference is that with that you can only change the anim of the mouth

**Comment by martonp96** [comment]
so what happens if you call taskPlayAnim with "mic_chatter", "mp_facial"?

**Comment by C0kkie** [comment]
The thing is then, you cant play a animation and change the anim of the mouth in the same time. If you would call taskPlayAnim with "mic_chatter", "mp_facial" then other animation would stop.

**Comment by tastydev** [comment]
Yes, this native is explicitly made for playing facial animations with normal/body animations concurrently, so i.e for voice solutions this will always be your way to go.
